modules/ROOT/pages/import-api-specification-design-center.adoc

```diff
@@ -0,0 +1,44 @@
+= Importing an API Specification from Design Center
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API Definition from Design Center into either into a new Mule Project, or an existing one.
```

Comment:
allows > enables
into either into > remove one **into**

Reply:
```suggestion
Anypoint Studio enables you to import an API definition from Design Center into either a new Mule project or an existing one.
```
I know it's tricky, but try to capitalize only things that are actual products or a category of features specific to a product (component, module, and so on): names. We tend to overcapitalize in tech docs, because that's what Dev does. When in doubt, lowercase.